Ambit/Source/Ambit/Mode/ConfigImportExport.spec.cpp

```diff
@@ -1017,4 +1017,133 @@ void ConfigImportExportSpec::Define()
             JsonContent = nullptr;
         });
     });
+
+    Describe("OnExportMap()", [this]()
+    {
+        BeforeEach([this]()
+        {
+            // Generate World
+            World = FAutomationEditorCommonUtils::CreateNewMap();
+
+            // Create Mode Settings
+            GLevelEditorModeTools().ActivateMode(FAmbitMode::EM_AmbitModeId);
+
+            FAmbitMode* AmbitMode = FAmbitMode::GetEditorMode();
```

Comment:
`OnExportMap()` and `OnExportGltf()` both use this check themselves, making it redundant for both. Unless we are needing it for the test cases, we can remove this and the line below.

Reply:
removed